build_msvc/README.md

```diff
@@ -17,7 +17,7 @@ Building with Visual Studio is an alternative to the Linux based [cross-compiler
 Prerequisites
 ---------------------
 To build [dependencies](../doc/dependencies.md) (except for [Qt](#qt)),
-the default approach is to use the [vcpkg](https://docs.microsoft.com/en-us/cpp/vcpkg) package manager from Microsoft:
+the default approach is to use the [vcpkg](https://vcpkg.io/en/index.html) package manager from Microsoft:
```

Comment:
nit: I think only `https://vcpkg.io/` could be enough, I think it points to `/en/index.html`.

```suggestion
the default approach is to use the [vcpkg](https://vcpkg.io/) package manager from Microsoft:
```

Reply:
✔️ Updated the link. Thank you very much!